internal/librarian/release_init.go

```diff
@@ -126,19 +129,42 @@ func (r *initRunner) run(ctx context.Context) error {
 		return err
 	}
 
-	commitInfo := &commitInfo{
+	var commits []*conventionalcommits.ConventionalCommit
+	for _, library := range r.state.Libraries {
+		commits = append(commits, library.Changes...)
+	}
+
+	// Only proceed with with the release if there are any releasable units
+	if getHighestChange(commits) == semver.None {
+		return nil
+	}
+
+	datetimeNow := formatTimestamp(time.Now())
+	branch := fmt.Sprintf("librarian-%s", datetimeNow)
+
+	commitInfo := &CommitInfo{
 		cfg:           r.cfg,
 		state:         r.state,
 		repo:          r.repo,
 		sourceRepo:    r.sourceRepo,
```

Comment:
I don't think source repos is being used here, could we do not assign the value here?